dime-coding-standards/checklists/tex/Reviewer Feedback Checklist

```diff
@@ -0,0 +1,154 @@
+		valign = center,
+		height = 0.3\textwidth,
+		after skip = 1cm]
+		#1
+	\end{tcolorbox}
+}
+
+\begin{document}
+	\begin{fullwidth}
+		
+		\titleBox{
+			\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Peer Code Review - Reviewer Feedback Checklist}} \\
+				\Large\textbf{{v1.0}}}
+		}
+
+        \dimeTextField{time}{2.5 cm}{Reviewer:} 
+
+        \dimeTextField{time}{2.5 cm} {Coder:}
+        
+        
+        This checklist lists important factors to consider while reviewing your code review partner's code package. Please fill this checklist, and submit it as an attachment when you submit \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed form.}
+        
+		\section*{Overview - First Impression}
+
+        Once you have confirmed that you received a package from your code review partner, please refer to the following list for basic recommended practices:
+        \begin{itemize}
+			\setlength\itemsep{-0.1em}		
+			
+			\item[] \dimeCheckBox{ Scripts for data cleaning, variable creation, and analysis are separate; and analysis scripts do not include any data processing, unless necessary for the creation of a table or graphic}
+			\item[] \dimeCheckBox{ Variable construction scripts include detailed comments about each variable created}
+			\item[] \dimeCheckBox{ Analysis scripts do not depend on having the results of other scripts in memory, except for the Master script}
+			\item[] \dimeCheckBox{ All scripts are well-commented and formatted, such that one can easily identify functional chunks of code and evaluate whether they correctly implement the econometric or statistical process described}
+			\item[] \dimeCheckBox{ Tables are output as .csv or .tex files or other raw text files when possible}
+		\end{itemize}
+	
+		\vspace{.6em}
+
+        \section*{Reproducibility}
+
+        \begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{Code ran perfectly on the first run \textbf{(Applicable only if data was shared with you.)}}
+			\item[] \dimeCheckBox{Outputs shared (if any) replicated perfectly on the first run}
+            \item[] \dimeCheckBox{If not, was able to determine the reason for why outputs changed}
+			\item[] \dimeCheckBox{Make note of any issues you faced in these steps. You will need to provide this information in \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed review summary form.}}
+			
+		\end{itemize}
+		
+        \vspace{.6em}
+
+       \section*{Ease of Use and Understanding}
+
+       
+        \begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{Understood the code from original documentation and README file}
+			\item[] \dimeCheckBox{Package needed additional information}
+            \item[] \dimeCheckBox{Understood the purpose of the code}
+			\item[] \dimeCheckBox{As a new RA joining this project right now, will need additional support to be able to contribute to this code}
```

Comment:
reword --> A new RA joining the project would be able to understand and contribute to the code